HacktoberfestProject.Web/Services/GithubService.cs

```diff
@@ -68,5 +69,67 @@ public async Task<ServiceResponse<IEnumerable<string>>> SearchOwners(string owne
                 ServiceResponseStatus = ServiceResponseStatus.Ok
             };
         }
+
+        public async Task<ServiceResponse<PrStatus?>> ValidatePrStatus(string owner, string repo, int id)
+        {
+            /*
+             * PRs count if:
+             *       Submitted in a repo with the hacktoberfest topic AND
+             *       during the month of October AND (
+             *         The PR is merged OR
+             *         The PR is labelled as hacktoberfest-accepted by a maintainer OR
+             *         The PR has been approved
+             *       )
+             */
+
+            var serviceResponse = new ServiceResponse<PrStatus?> { Content = PrStatus.Valid, ServiceResponseStatus = ServiceResponseStatus.Ok };
```

Comment:
Just checking: is there a possible issue with setting `PrStatus` to Valid by default? I see there's a catch-statement, but maybe we want to start with it being _not_ valid and then explicitly setting it to Valid if it is?